vuu-ui/packages/vuu-layout/src/layout-persistence/LayoutPersistenceManager.ts

```diff
@@ -35,12 +35,26 @@ export interface LayoutPersistenceManager {
    *
    * @returns Full JSON representation of the layout corresponding to the provided ID
    */
-  loadLayout: (id: string) => LayoutJSON;
+  loadLayout: (id: string) => Promise<LayoutJSON>;
 
   /**
    * Retrieves metadata for all existing layouts
    *
    * @returns an array of all persisted layout metadata
    */
-  loadMetadata: () => LayoutMetadata[];
+  loadMetadata: () => Promise<LayoutMetadata[]>;
+
+  /**
+   * Retrieves a temporary layout
```

Comment:
Might be worth expanding a bit on this documentation: what is a temporary layout and why are we saving them? I also wonder whether it would be better to call it something like _latest layout_ or _current layout_ instead of temporary?